docs/docs/user/migration-guide/v2-v3.md

```diff
@@ -0,0 +1,27 @@
+# Migrate from V2 to V3
+
+## [Make Kafka rest proxy & connect hosts default and improve schema registry config](https://github.com/bakdata/kpops/pull/354)
+
+The breaking changes target the `config.yaml` file:
+
+- The `schema_registry_url` is replaced with `schema_registry.enabled` (default `false`) and `schema_registry.url` (default `http://localhost:8081`).
+
+- `kafka_connect_host` is renamed to `kafka_connect_url` (default `http://localhost:8083`).
+- `kafka_rest_host` is renamed to `kafka_rest_url` (default `http://localhost:8082`).
+
+The environment variable name of these config fields changed respectively. Please refer to the [environment variables documentation page](../core-concepts/variables/environment_variables.md).
+
+Your `config.yaml` will change to:
+
+```diff
+  environment: development
+  brokers: "http://k8kafka-cp-kafka-headless.kpops.svc.cluster.local:9092"
```

Comment:
we should probably rename this to `kafka_brokers`, same as the associated env variable

Reply:
either this or omit `kafka` from the env var

Reply:
hm, but for example `KPOPS_CONNECT_URL` makes it sound like an entirely custom thing.

Reply:
I changed `KPOPS_CONNECT_URL` to `KPOPS_KAFKA_CONNECT_URL`

Reply:
shouldn't this be applied to brokers as well then, for consistency?